### PR TITLE
Verify <img ismap> server coordinate origin interop

### PR DESCRIPTION
**Pull request by travisleithead**
In relation to: https://github.com/w3c/html/issues/492

Firefox/Chrome follow the spec. Edge is a little wonky... I hear webkit is also not quite aligned... perhaps someone can run these tests and let me know?

Tests:
> If the click event was a real pointing-device-triggered click event on the img element, then let x be the distance in CSS pixels from the left edge of the image’s left border, if it has one, or the left edge of the image otherwise, to the location of the click, and let y be the distance in CSS pixels from the top edge of the image’s top border, if it has one, or the top edge of the image otherwise, to the location of the click. Otherwise, let x and y be zero.

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/209945592)

# Firefox (nightly channel) #
Testing web-platform-tests at revision e040b9b46e64c8029341c1c8a403f1b2f2a1b2ba
Using browser at version BuildID 20170306110339; SourceStamp 966464a68a2cb3ca1125808e34abb5c1d34e3797
Starting 10 test iterations
No tests run.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/209945593)

# Chrome (unstable channel) #
Testing web-platform-tests at revision e040b9b46e64c8029341c1c8a403f1b2f2a1b2ba
Using browser at version 58.0.3029.14 dev
Starting 10 test iterations
No tests run.

**Comment by zcorpan** [comment]
Result in Safari TP Release 25 (Safari 10.2, WebKit 12604.1.8.1.2)

```
Fail	Coordinates within the content box of an image map have origin of the context box	assert_true: The x coordinate [152] must be greater-than (or equal-to) 0 and less-than (or equal-to) 96 expected true got false
http://web-platform.test:8000/html/semantics/embedded-content/the-img-element/ismap/img-ismap-coordinates-manual.html:66:20
step@http://web-platform.test:8000/resources/testharness.js:1409:30
testInit@http://web-platform.test:8000/html/semantics/embedded-content/the-img-element/ismap/img-ismap-coordinates-manual.html:58:16
Fail	Coordinates within the margin/padding (top-left) of the image map are clamped to zero	assert_true: The x coordinate [68] must be greater-than (or equal-to) 0 and less-than (or equal-to) 0 expected true got false
http://web-platform.test:8000/html/semantics/embedded-content/the-img-element/ismap/img-ismap-coordinates-manual.html:66:20
step@http://web-platform.test:8000/resources/testharness.js:1409:30
testInit@http://web-platform.test:8000/html/semantics/embedded-content/the-img-element/ismap/img-ismap-coordinates-manual.html:58:16
Fail	Coordinates within the margin/padding (bottom-right) of the image map have origin in the content box	assert_true: The x coordinate [214] must be greater-than (or equal-to) 97 and less-than (or equal-to) 146 expected true got false
http://web-platform.test:8000/html/semantics/embedded-content/the-img-element/ismap/img-ismap-coordinates-manual.html:66:20
step@http://web-platform.test:8000/resources/testharness.js:1409:30
testInit@http://web-platform.test:8000/html/semantics/embedded-content/the-img-element/ismap/img-ismap-coordinates-manual.html:58:16
```

**Comment by zcorpan** [comment]
Thanks for putting together a test case! I filed a bug for WebKit: https://bugs.webkit.org/show_bug.cgi?id=169548

**Comment by zcorpan** [comment]
This needs to be rebased on latest master.

**Comment by cdumez** [comment]
Any update on this?

File: html/semantics/embedded-content/the-img-element/ismap/img-ismap-coordinates-manual.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      // Validate the last test's results...
+      test.step(function () {
+        var i = url.indexOf("?");
+        assert_not_equals(i, -1);
+        var coordsStr = url.substr(i+1);
+        var i = coordsStr.indexOf(',');
+        assert_not_equals(i, -1);
+        var x = parseFloat(coordsStr.substring(0, i));
+        var y = parseFloat(coordsStr.substring(i+1));
+        assert_true((x >= minXY) && (x <= maxXY), "The x coordinate ["+x+"] must be greater-than (or equal-to) " + minXY + " and less-than (or equal-to) " + maxXY);
```
Comment (zcorpan) [inline html/semantics/embedded-content/the-img-element/ismap/img-ismap-coordinates-manual.html]: These can use `assert_greater_than_equal` and `assert_less_than_equal` instead.

**Comment by travisleithead** [comment]
Will try to refresh this today...

**Comment by travisleithead** [comment]
Rebased and force-pushed.
Added the recommended change from @zcorpan's review.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/234146118)

# Firefox (nightly) #
Testing web-platform-tests at revision 804265f905a25dbd4c207ecf181f2f1fe25afd10
Using browser at version BuildID 20170519133342; SourceStamp 8e98dab5054dd093a37ba20c62cf0523e484cfbd
Starting 10 test iterations
No tests run.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/234146119)

# Sauce (safari) #
Testing web-platform-tests at revision 804265f905a25dbd4c207ecf181f2f1fe25afd10
Using browser at version 10.0
Starting 10 test iterations
No tests run.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/234146125)

# Chrome (unstable) #
Testing web-platform-tests at revision 804265f905a25dbd4c207ecf181f2f1fe25afd10

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/234146126)

# Sauce (MicrosoftEdge) #
Testing web-platform-tests at revision 804265f905a25dbd4c207ecf181f2f1fe25afd10

**Review by zcorpan — APPROVED** [review]

**Pull request merged**